src/Schema/Directives/Nodes/EnumDirective.php

```diff
@@ -54,7 +55,7 @@ public function resolveNode(NodeValue $value)
      *
      * @return string
      */
-    protected function safeDescription($description = '')
+    protected function safeDescription(string $description = ''): string
```

Comment:
I assume this will effect things elsewhere, but if `null` is passed in this throws an error (since it's forcing a string to be passed). This gets called from line 45 w/ `$field->description` which can return `null`.

Reply:
The default `= ''` actually catches that. See https://3v4l.org/prfFU

Reply:
Add the following enum to the schema [for this test](https://github.com/nuwave/lighthouse/blob/master/tests/Integration/Schema/Directives/Args/QueryFilterDirectiveTest.php#L30-L38)

```graphql
enum Role {
    ADMIN @enum(value: "ADMIN")
    EMPLOYEE @enum(value: "EMPLOYEE")
}
```

When I do, I get the error:

```
TypeError: Argument 1 passed to Nuwave\Lighthouse\Schema\Directives\Nodes\EnumDirective::safeDescription() must be of the type string, null given, called in /lighthouse-mll/src/Schema/Directives/Nodes/EnumDirective.php on line 45
```